.travis.yml

```diff
@@ -38,4 +38,4 @@ script:
   - py.test
   # test that the py2cyto conda environment can be created as well
   - source deactivate
-  - conda env create -f visualization_scripts/minimal_env.yml
+  - conda env create -f workflow/minimal_env.yml
```

Comment:
Can we now run`workflow/main.py` in this environment during the Travis CI tests?

Reply:
Yes, this should resolve the test previously failing